zarr/tests/test_storage_v3.py

```diff
@@ -235,6 +254,46 @@ def test_rename_nonexisting(self):
             with pytest.raises(NotImplementedError):
                 store.rename('a', 'b')
 
+    def test_get_partial_values(self):
```

Comment:
This is a nice test case. One thing that still looks missing here is also including a range that has `None` for the second value? (Based on the `StorageTransformer` class it seems that is also supported)

Reply:
Good point, I added this :+1: